index.html

```diff
@@ -8,12 +8,12 @@
 	  <meta http-equiv="X-UA-Compatible" content="IE=edge">
 	  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-	  <title>100 Days</title>
+    <title>{{ site.title }}</title>
     <meta name="description" content="{{ site.description }}">
 
 	  <link rel="stylesheet" href="/assets/main.css">
-	  <link rel="canonical" href="http://localhost:4000/">
-	  <link rel="alternate" type="application/rss+xml" title="100 days" href="/feed.xml">
+    <link rel="canonical" href="{{ site.production_url }}{{ page.url | replace:'index.html',''}}">
```

Comment:
nice, tho again not sure if a separate production_url is needed! 

But eventually this probably for the template moreso than a static index?